sys/include/net/gnrc/netif/conf.h

```diff
@@ -31,18 +31,6 @@
 extern "C" {
 #endif
 
-/**
- * @brief   Single interface optimizations
- *
- *          Define to 1 to allow GNRC optimizations when only one interface
- *          is available.
- *
- * @note    This MUST NOT be enabled if there's more than one interface.
- */
```

Comment:
I'm afraid we have no way of documenting pseudo-modules :(

Reply:
You could move it to the detailed description in `net/gnrc/netif.h`.

Reply:
Ok.